packages/assets/extract-svgs.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Extracts icons from VADS icon sprite as single svgs
+ * Adapted from https://gist.github.com/mhmdnsr/18343cceb12858c8b148ef31d1e5a32b
+ */
+
+import * as fs from 'fs'
+import { parse } from 'svg-parser'
+import { toHtml } from 'hast-util-to-html'
+
+fs.readFile(
+  '../../node_modules/@department-of-veterans-affairs/component-library/dist/img/sprite.svg',
+  'utf8',
+  function (err, contents) {
+    const parsed = parse(contents)
+    const symbols = parsed.children[0].children
+    let length = 0
+    symbols.forEach((symbol) => {
+      const name = symbol.properties.id
+      symbol.tagName = 'svg'
+      let newIcon = toHtml(symbol)
+      fs.writeFile(`icons/vads/${name}.svg`, newIcon, () => {
```

Comment:
Maybe a question we need to loop Jessica in on, but should we try to de-duplicate here with the existing icons we have? Removing name conflicts from either ours or VADS seems maybe easier with the script than on the component side where we either drop icons on the floor or need separate names for them.

Reply:
Looking through the icons, this would be somewhat problematic with `home` since they only have one and that would make the nav bar icon change in unintended ways (not just go filled to outline, but a totally different house).

Reply:
Per stand-up, sounds like we intend to mostly clean up our icons and will have an updated `home` icon that is an outline form of VADS/Material icons. So we do not need to worry about automatic de-duplicating as we'll handle it soonTM on follow-up tickets.